app/src/main/java/org/hwyl/sexytopo/control/graph/GraphView.java

```diff
@@ -1394,26 +1386,39 @@ public void centreViewOnSurveyPoint(Coord2D point) {
         viewpointOffset = new Coord2D(x, y);
     }
 
+    private void drawDashedLine(Canvas canvas,
+                                Coord2D start, Coord2D end,
+                                float dashLength,
+                                Paint paint) {
 
-    private void drawLine(Canvas canvas, Coord2D start, Coord2D end, Paint paint, int alpha) {
-        paint.setAlpha(alpha);
-        canvas.drawLine(
-                (float)(start.x), (float)(start.y),
-                (float)(end.x), (float)(end.y),
-                paint);
-    }
+        double lineLength = Space2DUtils.getDistance(start, end);
+        int dashes = (int)(lineLength / dashLength);
 
+        Coord2D direction = end.minus(start).normalize();
```

Comment:
Nice elegant way of doing it, but I'm a bit surprised to find Cartesian co ords being used as vectors. I guess they are technically vectors...